src/JoomlaBrowser.php

```diff
@@ -1099,26 +1042,195 @@ public function createCategory($title, $extension = '')
 		{
 			$extension = '&extension=' . $extension;
 		}
-
 		$this->amOnPage('administrator/index.php?option=com_categories' . $extension);
-
 		$this->waitForElement(['class' => 'page-title']);
 		$this->checkForPhpNoticesOrWarnings();
-
 		$this->debug('Click new category button');
 		$this->click($this->locator->adminToolbarButtonNew);
-
 		$this->waitForElement(['class' => 'page-title']);
-
 		$this->checkForPhpNoticesOrWarnings();
 		$this->fillField(['id' => 'jform_title'], $title);
-
 		$this->debug('Click new category apply button');
 		$this->click($this->locator->adminToolbarButtonApply);
-
 		$this->debug('see a success message after saving the category');
-
 		$this->see('Category saved', ['id' => 'system-message-container']);
 		$this->checkForPhpNoticesOrWarnings();
 	}
+
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
```

Comment:
wrong Since, remember these changes are for 4.0 Version, and your PR is also on the 4.0 branch